README.md

```diff
@@ -1,16 +1,52 @@
 # JNI bindings for Roc Toolkit
 
 [![Build Status](https://travis-ci.org/roc-streaming/roc-java.svg?branch=master)](https://travis-ci.org/roc-streaming/roc-java)
-[![Android release](https://img.shields.io/bintray/v/roc-streaming/maven/roc-android?color=blue&label=android)](https://bintray.com/roc-streaming/maven/roc-android/_latestVersion)
+[![Android release](https://img.shields.io/bintray/v/roc-streaming/maven/roc-android?color=blue&label=aar)](https://bintray.com/roc-streaming/maven/roc-android/_latestVersion)
 
-_Work in progress!_
+This library provides JNI bindings for [Roc Toolkit](https://github.com/roc-streaming/roc-toolkit), a toolkit for real-time audio streaming over the network.
 
-## Dependencies
+The bindings can be used in Java, Kotlin, and other JVM-based languages.
 
-You will need to have libroc and libroc-devel (headers) installed. Refer to official build [instructions](https://roc-streaming.org/toolkit/docs/building.html) on how to install libroc. There is no official distribution for any OS as of now, you will need to install from source.
+Android support included!
 
-## Quickstart
-_At the moment we release only android library_
+## About Roc
+
+Compatible senders and receivers include:
+
+* [command-line tools](https://roc-streaming.org/toolkit/docs/running/command_line_tools.html)
+* [PulseAudio modules](https://roc-streaming.org/toolkit/docs/running/pulseaudio_modules.html)
+* [C library](https://roc-streaming.org/toolkit/docs/api.html)
+* [Go bindings](https://github.com/roc-streaming/roc-go/)
+
+Key features:
+
+* real-time streaming with guaranteed latency;
+* restoring lost packets using Forward Erasure Correction codes;
+* converting between the sender and receiver clock domains;
+* CD-quality audio;
+* multiple profiles for different CPU and latency requirements;
+* portability;
+* relying on open, standard protocols.
+
+## Documentation
+
+Documentation for the Java API is not available online yet. You can find it in the source code as javadoc comments.
+
+Documentation for the C API can be found [here](https://roc-streaming.org/toolkit/docs/api.html).
+
+## Versioning
+
+Java bindings and the C library both use [semantic versioning](https://semver.org/).
+
+The first two components (major and minor) of the bindings and the C library versions correspond to each other. The third component (patch) is indepdendent.
+
+**Bindings are compatible with the C library if its major version is the same, and minor version is the same or higher.**
+
+For example, version 1.2.3 of the bindings would be compatible with 1.2.x and 1.3.x, but not with 1.1.x (minor version is lower) or 2.x.x (major version is different).
```

Comment:
As far as I understand the next version of the toolkit will be `0.2.0`. Will it contain breaking changes? 
I mean will the current version of bindings (0.1.0 if we make a release) be compatible with the `0.2.0` toolkit?

Reply:
Good catch. If it were transition from 1.1.x to 1.2.x, semantic version would allow adding new features and forbid breaking changes.

However while the major version is zero (0.x.x), semantic version allow breaking changes in any release. Zero version is considered totally unstable.

We use a bit stricter approach, and while the major version is zero, we break things only when we increment minor version. And yes, 0.2.0 will break API and we'll have to update bindings.

I'll add this to README.

Reply:
> I'll add this to README.

Done!